modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -407,659 +406,470 @@
-                  ],
-                  "decision": {
-                    "statuses": [
-                      "none",
-                      "accepting",
-                      "declining"
-                    ]
-                  }
-                },
-                "page": {
-                  "number": 2,
-                  "size": 3
-                },
-                "sort": {
-                  "field": "createdAt",
-                  "order": "asc"
-                }
-              }
-            },
-            "example": {
-              "filter": {
-                "poaCodes": [
-                  "083",
-                  "002",
-                  "003",
-                  "065",
-                  "074",
-                  "022",
-                  "091",
-                  "070",
-                  "097",
-                  "077",
-                  "1EY",
-                  "6B6",
-                  "862",
-                  "9U7",
-                  "BQX"
-                ],
-                "decision": {
-                  "statuses": [
-                    "none",
-                    "accepting",
-                    "declining"
-                  ]
-                }
-              },
-              "page": {
-                "number": 2,
-                "size": 3
-              },
-              "sort": {
-                "field": "createdAt",
-                "order": "asc"
-              }
-            }
-          }
+
         ],
         "responses": {
           "200": {
             "description": "Search results",
             "content": {
               "application/json": {
-                "example": {
-                  "metadata": {
-                    "totalCount": 121,
-                    "query": {
-                      "filter": {
-                        "poaCodes": [
-                          "083",
-                          "002",
-                          "003",
-                          "065",
-                          "074",
-                          "022",
-                          "091",
-                          "070",
-                          "097",
-                          "077",
-                          "1EY",
-                          "6B6",
-                          "862",
-                          "9U7",
-                          "BQX"
-                        ],
-                        "decision": {
-                          "statuses": [
-                            "none",
-                            "accepting",
-                            "declining"
-                          ]
-                        }
-                      },
-                      "page": {
-                        "number": 2,
-                        "size": 3
-                      },
-                      "sort": {
-                        "field": "createdAt",
-                        "order": "asc"
-                      }
-                    }
+                "example": [
```

Comment:
thinking I'd trim the cassette that was used to build this. 16 decisions / 300+ lines of code for the example is a good bit. I think we could get away with 2 or 3 examples, but 🤷

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/19048/commits/5fc900c7fdeb4cc72482195f2a5f707dfa7627bc